users/kgarg.json

```diff
@@ -0,0 +1,6 @@
+{
+  "copyright": "Kunal Garg",
+  "url": "http://kunalgarg2100.github.io",
+  "email": "kunalgarg2100@gmail.com",
+  "gravatar": true,
```

Comment:
in order for the tests for https://travis-ci.org/remy/mit-license/builds/334868809 to pass, I think you want to remove the trailing `,` from this line